rules_bazel/java/java_diff_test.bzl

```diff
@@ -0,0 +1,85 @@
+def _junit_output_impl(ctx):
+    test_class_name = ctx.attr.test_class_name
+    inputs = ctx.files.srcs
+    output = ctx.outputs.output
+    test_runner = ctx.executable.test_runner
+
+    command = """
+    mkdir local_tmp  
+    TEST_OUTPUT_HOME="$(pwd)/local_tmp" \
+    {test_runner_path} $@ &&
+    cd local_tmp 
+    zip -r ../{output} .
```

Comment:
`zip -r -j {output} local_tmp/*` instead of `cd`?

Reply:
Unfortunately in `local_tmp` we have `com/google/api/generator/.../File1.golden` and `-j` option will flatten the path and only zip `File1.golden` file without the folders.

Reply:
Done.